specific_use_cases/DFU_work_in_progress/INTRO_annual_consumption_model.ipynb

```diff
@@ -427,7 +423,7 @@
    "source": [
     "### 4a) Decide which input data you want to use \n",
     "\n",
-    "You can use the data within the demand forecast zone, which we retrieved in **step 2b**. Or, you can use the closest grid cell to the weather station, which we computed in **step 1b**. We do not recommend using the aggregated DFZ data calculated in step 3b, as aggregating the data prior to computing HDD and CDD may remove some critical information about the weather extremes. You can comment out whichever method you don't want to use. We've chosen to show the analysis with the DFZ data, but if you want to use the closest grid cell data, just comment out the DFZ cells and uncomment the closest grid cells. Note, the closest grid cell resampling will take a few minutes! <br><br>Depending on the input data, we will also set a new variable defining the number of grid cells. This will of course be just 1 for the closest grid cell method; for the DFZ data, however, this value will change depending on the size of the DFZ before aggregating. This information is used to compute the annual aggregate HDD and CDD in step 4c."
+    "You can use the data within the demand forecast zone, which we retrieved in **step 2b**. Or, you can use the closest grid cell to the weather station, which we computed in **step 1b**. We do not recommend using the aggregated DFZ data calculated in step 3b, as aggregating the data prior to computing HDD and CDD may remove some critical information about the weather extremes. You can comment out whichever method you don't want to use. We've chosen to show the analysis with the DFZ data, but if you want to use the closest grid cell data, just comment out the DFZ cells and uncomment the closest grid cells. Note, the closest grid cell resampling will take a few minutes! <br><br>Depending on the input data, we will also set a new variable defining the number of grid cells. This will be just 1 for the closest grid cell method; for the DFZ data, however, this value will change depending on the size of the DFZ before aggregating. This information is used to compute the annual aggregate HDD and CDD in step 4c. Lastly, we provide 2 options for determining the [daily mean](https://www.weather.gov/key/climate_heat_coolhttps://www.weather.gov/key/climate_heat_cool) for which we calculate degree days against. "
```

Comment:
link pasted twice, redirects

Reply:
Also in the weather.gov page linked here, I found this sentence useful for describing why we calculate HDD/CDD: "Degree days are based on the assumption that when the outside temperature is 65°F, we don't need heating or cooling to be comfortable." 

This notebook seems to be pretty specific and assumes that the user knows why they want to calculate HDD/CDD, and wants that threshold temperature to be flexible-- is it worth including something like this in the text introducing Step 4 as a reminder of why the user might set a certain temperature threshold ?

Reply:
Great catch, fixed! And yes, I think that would be super useful context to include in the markdown!